layouts/partials/hero-body.html

```diff
@@ -17,8 +17,8 @@
                     {{ . }}
                 </div>
                 {{ end }}
-                <div class="hero-cta">
-                    <button class="cta-button">{{ $buttonText }}</button>
+                <div class="hero-cta" href="/learn">
```

Comment:
the href attribute on the div here won't do anything, but the code below (the anchor wrapping the button) looks great!

Reply:
Good catch, that was a leftover from trying to figure out what worked. Fixed now.